docs/python/inference/notebooks/onnxruntime-tvm-tutorial.ipynb

```diff
@@ -302,29 +302,28 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
+     "name": "stderr",
      "output_type": "stream",
      "text": [
-      "STVM ep options:\n",
+      "[13:00:57] /home/agladyshev/workspace/onnxruntime/onnxruntime/core/providers/tvm/tvm_execution_provider.cc:480: TVM EP options:\n",
```

Comment:
@KJlaccHoeUM9l - this should be cleaned up to just "TVM EP"!

Reply:
I made the cleanup; I'll work with Xavier to get this merged, thanks Valery

Reply:
I saw. It is pitty that trick from Xavier does not work. I'm not sure it looks like the text from ipynb is autogenerated, I asked Alexey to rechecked it. Let's merge this PR, if we find some issue related to the renaming we will fix it further